packages/reactive-element/src/reactive-element.ts

```diff
@@ -643,7 +643,7 @@ export abstract class ReactiveElement
     // still overwrite it
     if (!options.noAccessor && !this.prototype.hasOwnProperty(name)) {
       const key = DEV_MODE
-        ? Symbol(`${String(name)} (@property() cache)`)
+        ? Symbol.for(`${String(name)} (@property() cache)`)
```

Comment:
Add a comment about why you're using Symbol.for()

Reply:
Done!